core/src/main/java/fr/sncf/osrd/api/SignalingSimulator.kt

```diff
@@ -9,17 +10,33 @@ import fr.sncf.osrd.signaling.impl.SignalingSimulatorImpl
 import fr.sncf.osrd.signaling.tvm300.*
 import fr.sncf.osrd.signaling.tvm430.*
 
+val signalingSystemNameToCost =
```

Comment:
```suggestion
val signalingSystemToCost =
```

Or even just `signalingSystemCost`

Reply:
I also got rid of the `id` on the way: https://github.com/OpenRailAssociation/osrd/pull/10857/commits/2789daa024e4b073e32f6858d8b7c924f7f46f65